vespaclient-container-plugin/src/test/java/com/yahoo/document/restapi/resource/StreamingJsonLinesResponseTest.java

```diff
@@ -252,36 +252,34 @@ void provided_tensor_options_are_used_when_rendering_put() throws IOException {
     }
 
     @Test
-    void trace_is_rendered_as_own_line() throws IOException {
+    void request_trace_is_rendered_as_own_line() throws IOException {
         var f = new Fixture();
         var trace = new Trace(9);
         trace.trace(7, "Poirot's deductions", false);
         trace.getRoot().addChild(new TraceNode().setStrict(false)
                 .addChild("The butler did it")
                 .addChild("Weapon was a garden gnome"));
-        f.jsonlResponse.writeTrace(trace);
-        // TODO the nested "trace" objects aren't beautiful; reconsider format for JSONL.
-        //  Since trace nodes may technically output "message" objects at any level, need a wrapper.
+        f.jsonlResponse.writeRequestTrace(trace);
         String expected = """
-                {"trace":{"trace":[{"message":"Poirot's deductions"},{"fork":[{"message":"The butler did it"},{"message":"Weapon was a garden gnome"}]}]}}
+                {"requestTrace":{"trace":[{"message":"Poirot's deductions"},{"fork":[{"message":"The butler did it"},{"message":"Weapon was a garden gnome"}]}]}}
```

Comment:
My suggestions (in random order): `operationTrace`, `documentApiTrace`, `documentTrace`.
The naming `requestTrace` could indicate that it's specific to HTTP.